cmd/cloudshell_open/main.go

```diff
@@ -214,24 +214,36 @@ func run(c *cli.Context) error {
 	serviceName = tryFixServiceName(serviceName)
 
 	image := fmt.Sprintf("gcr.io/%s/%s", project, serviceName)
-	fmt.Println(infoPrefix + " FYI, running the following command:")
-	cmdColor.Printf("\tdocker build -t %s %s\n", parameter(image), parameter("."))
 
 	end = logProgress(fmt.Sprintf("Building container image %s", highlight(image)),
 		fmt.Sprintf("Built container image %s", highlight(image)),
 		"Failed to build container image.")
-	err = build(appDir, image)
-	end(err == nil)
+
+	exists, err := dockerFileExists(appDir)
 	if err != nil {
 		return err
 	}
 
+	if exists {
+		fmt.Println(infoPrefix + " FYI, running the following command:")
+		cmdColor.Printf("\tdocker build -t %s %s\n", parameter(image), parameter("."))
+		err = dockerBuild(appDir, image)
+	} else {
+		fmt.Println(infoPrefix + " FYI, running the following command:")
```

Comment:
wdyt of adding an always-visible info here when this path is taken:

    fmt.Println(infoPrefix + " Attempting to build this application with Cloud Native Buildpacks (buildpacks.io)...")

we can also add the same info to the dockerBuild block too

    fmt.Println(infoPrefix + " Attempting to build this application with its Dockerfile...")